pkg/operator/controllers/guardrails/guardrails_config.go

```diff
@@ -0,0 +1,120 @@
+package guardrails
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"embed"
+	"strings"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	"github.com/Azure/ARO-RP/pkg/operator/controllers/guardrails/config"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+const (
+	ControllerName               = "GuardRails"
+	controllerEnabled            = "aro.guardrails.enabled"
+	controllerNamespace          = "aro.guardrails.namespace"
+	controllerManaged            = "aro.guardrails.deploy.managed"
+	controllerPullSpec           = "aro.guardrails.deploy.pullspec"
+	controllerManagerRequestsCPU = "aro.guardrails.deploy.manager.requests.cpu"
+	controllerManagerRequestsMem = "aro.guardrails.deploy.manager.requests.mem"
+	controllerManagerLimitCPU    = "aro.guardrails.deploy.manager.limit.cpu"
+	controllerManagerLimitMem    = "aro.guardrails.deploy.manager.limit.mem"
+	controllerAuditRequestsCPU   = "aro.guardrails.deploy.audit.requests.cpu"
+	controllerAuditRequestsMem   = "aro.guardrails.deploy.audit.requests.mem"
+	controllerAuditLimitCPU      = "aro.guardrails.deploy.audit.limit.cpu"
+	controllerAuditLimitMem      = "aro.guardrails.deploy.audit.limit.mem"
+
+	controllerValidatingWebhookFailurePolicy = "aro.guardrails.validatingwebhook.managed"
+	controllerValidatingWebhookTimeout       = "aro.guardrails.validatingwebhook.timeoutSeconds"
+	controllerMutatingWebhookFailurePolicy   = "aro.guardrails.mutatingwebhook.managed"
+	controllerMutatingWebhookTimeout         = "aro.guardrails.mutatingwebhook.timeoutSeconds"
+
+	controllerReconciliationMinutes     = "aro.guardrails.reconciliationMinutes"
+	controllerPolicyManagedTemplate     = "aro.guardrails.policies.%s.managed"
+	controllerPolicyEnforcementTemplate = "aro.guardrails.policies.%s.enforcement"
+
+	defaultNamespace = "openshift-azure-guardrails"
+
+	defaultManagerRequestsCPU = "100m"
+	defaultManagerLimitCPU    = "1000m"
```

Comment:
Generally, limits on CPU (a compressible resource) are considered bad practice in k8s, in fact, you'll notice that most operators in OCP do not ship with CPU limits, and only workloads that are capable of failing 100% without disruption to the cluster set limits (e.g. cleanup tasks). ref: `oc get po -o custom-columns="Name:metadata.name,CPU-limit:spec.containers[*].resources.limits.cpu" -A` on the shared cluster.

For this workload specifically, I could eventually see this manifest as a large portion of requests timing out due to not having enough CPU cycles to handle a large number of API requests, with the Node this is being hosted on being at 25% CPU utilization on a 4-core machine.

So, to get to the point: When we've finally matured Gatekeeper with enforcement, what would a large portion of requests to gatekeeper timing out look like to a customer? My guess is (unwarranted) 401s, but it could be something else?